### PR TITLE
fixing menu icon

### PR DESCRIPTION
**Pull request by prateekbh**
migrates to `mdc-web` 0.23.0

### CODE REVIEW

File: Toolbar/Toolbar.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ class ToolbarIcon extends MaterialComponent {
     super();
     this.componentName = "toolbar__icon";
     if (props.menu) {
-      this.componentName += "--menu";
+      this.componentName += "-menu";
```
Comment (filoozom) [inline Toolbar/Toolbar.jsx]: The final output should be ```mdc-toolbar__menu-icon```, not ```mdc-toolbar_icon-menu```.

https://github.com/material-components/material-components-web/blob/master/CHANGELOG.md#breaking-changes

Comment (filoozom) [reply Toolbar/Toolbar.jsx]: Maybe a new component would be cleaner than completely changing ```componentName```?

**Comment by prateekbh** [comment]
Thanks for catching that 👍

**Comment by filoozom** [comment]
Still not right though 😛 
Should be ```this.componentName = "toolbar__menu-icon";```. Now it's ```mdc-toolbar__icon-menu-icon```.

**Comment by prateekbh** [comment]
@filoozom what would i do without ur reviews 😛 . Thanks for taking care of this

**Pull request merged**